rakefile.rb

```diff
@@ -182,11 +183,17 @@ def examples
           sh "cross build --release --bin #{name} --target #{target_arch}"
           cp "target/#{target_arch}/release/#{name}", target_dir
         end
-        create_arch_package(target_arch, target_dir, dir, registry, package_config)
+        create_arch_package_sextant(dir, registry, target_arch)
+        # create_arch_package(target_arch, target_dir, dir, registry, package_config)
       end
     end
   end
 
+  def create_arch_package_sextant(container_src, out_dir, target_arch)
+    puts "target arch was: #{target_arch}"
+    # sh "target/release/sextant pack -d #{container_src} -o #{out_dir} -k #{KEY_FILE}"
+  end
+
```

Comment:
I wouldn't change this in the pr.
If the shellscript is to be used, get rid of the rake task